src/lib/plugin.ts

```diff
@@ -10,6 +10,7 @@ import type esbuild from 'esbuild';
 const NAME = 'node-modules-polyfills';
 
 export interface NodePolyfillsOptions {
+	fallback?: 'empty' | 'none';
```

Comment:
I went with this API so that it's more future proof, as opposed to `emptyFallback: true` or something similar. This way we can introduce different fallback behaviour in the future if needed, e.g. `fallback: 'error'` or `fallback: (args) => { /* some custom logic... */ }`.